lib/bundle.js

```diff
@@ -218,7 +221,9 @@ exports.build = function(expression, fileName, opts) {
   opts = opts || {};
   fileName = fileName || path.resolve(config.pjson.baseURL, 'build.js');
 
-  function build() {
+  function build(newBuilder) {
+    var latestBuilder = newBuilder || systemBuilder;
```

Comment:
It would be nice to keep this function consistent - perhaps let's make `builder` a required argument always for the `build` function, and make sure all calls to `build` provide the builder instance?